scripts/github_runner/start_runners.sh

```diff
@@ -50,16 +50,15 @@ docker pull vhiveease/cri_test_runner
 
 
 
-for number in $(seq 1 $1)
-do
-    case "$4" in
-    "integ")
-        
-        if [ "$5" == "restart" ]; then
-            docker container stop $(docker ps --format "{{.Names}}" | grep integration_test-github_runner)
-            docker container rm $(docker ps -a --format "{{.Names}}" | grep integration_test-github_runner)
-        fi
-        
+case "$4" in
```

Comment:
@shyamjesal can you please replace all these 1, 2, 3, 4, 5 with named variables?

Reply:
actually let's do it in the other PR.